extensions/amp-ad-exit/0.1/test-e2e/test-amp-ad-exit.js

```diff
@@ -23,9 +24,15 @@ describes.endtoend(
   },
   env => {
     let controller;
+    let requestBank;
 
-    beforeEach(async () => {
+    beforeEach(() => {
       controller = env.controller;
+      requestBank = new RequestBankE2E('http://localhost:8000', 'ampadexit');
```

Comment:
do we need these 2 params?

Reply:
OK, I saw `ampadexit` is used in the HTML file. we had an issue in integration test when multiple tests run in parallel in different browsers (on SauceLab), they are sharing the so server endpoint hence causing interferences to each others.

I don't know about our e2e tests setup, @estherkim can multiple browsers run the same test in parallel with the same test server? If not, it might not be an issue here.

Reply:
The second one is optional. It is needed when it is difficult to inject the link to the doc.

Reply:
An e2e test runs in multiple browsers but in series. So having the same test server should be fine.